pkg/log/messages/error_messages.go

```diff
@@ -54,4 +54,5 @@ const CSPFK036E string = "CSPFK036E Variable ID '%s' is not in the format '<acco
 const CSPFK037E string = "CSPFK037E Failed to parse DAP/Conjur variable IDs"
 
 // General
+const CSPFK039E string = "CSPFK039E Secrets Provider for Kubernetes failed to update Kubernetes Secrets"
 const CSPFK038E string = "CSPFK038E Retransmission backoff exhausted"
```

Comment:
is this still used?

Reply:
yes if after 3 retries, the user should hit this log

Reply:
I used `printErrorAndExit` instead of log.error which is wrong. I will update this

Reply:
jumped all the logic up to after count = retry count so the deleting of accessToken still takes place